polkadot/xcm/src/v4/junction.rs

```diff
@@ -131,10 +138,6 @@ pub enum NetworkId {
 	Polkadot,
 	/// The Kusama canary-net Relay-chain.
 	Kusama,
-	/// The Westend testnet Relay-chain.
-	Westend,
-	/// The Rococo testnet Relay-chain.
-	Rococo,
```

Comment:
you cannot change this struct for a live version - this struct is part of the XCM ABI, it influences the encoding, changing it like this will break all chains using XCM